styles.css

```
@@ -19,15 +22,20 @@
   border: 5px;
   border-radius: 5px;
   border-color: rgb(73, 10, 51);
```

You can combine the border declarations:

```suggestion
  border: 5px solid rgb(73, 10, 51);
  border-radius: 5px;
```